src/net-questdb-client/Ingress/Senders/TcpSender.cs

```diff
@@ -0,0 +1,314 @@
+
+using System.Buffers.Text;
+using System.Net.Security;
+using System.Net.Sockets;
+using Org.BouncyCastle.Asn1.Sec;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.Math;
+using Org.BouncyCastle.Security;
+using QuestDB.Ingress.Buffers;
+using QuestDB.Ingress.Enums;
+using QuestDB.Ingress.Utils;
+using Buffer = QuestDB.Ingress.Buffers.Buffer;
+using ProtocolType = QuestDB.Ingress.Enums.ProtocolType;
+
+
+namespace QuestDB.Ingress.Senders;
+
+internal class TcpSender : ISender
```

Comment:
Can we wrap around obsolete Tcp Sender instead of code duplication?

Reply:
Can we then the other way around, implement the old sender around as calls to new TCP sender?